docker/importer/importer.py

```diff
@@ -502,7 +507,8 @@ def _process_updates_rest(self, source_repo: osv.SourceRepository):
 
     replace_importer_log(storage.Client(), source_repo.name,
                          self._public_log_bucket, import_failure_logs)
-
+    if ignore_last_import_time:
```

Comment:
Let's move this up to 463 to keep the behavior similar to the bucket import function (and add another source_repo.put()). We want to have ignore last import time set back to false even if there's an exception happening between 463 and 513.

Reply:
done :) 👍🏻